_posts/2019-10-5-the-manual-tester-becoming-the-best-QA-asset-for-your-team.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title:  "The Manual Tester: Becoming the Best QA Asset For Your Team"
+date: 2019-10-5 11:00:00
+categories: ["software-development", "software-quality"]
+author: abizzinotto
+---
+
+No app is flawless. We all know that. Quality Assurance is an important part of any software development process and the better the tester, the higher quality the software that gets deployed to production.
+
+But… how to be a better manual tester? Applications have evolved greatly and are becoming more and more powerful, but the manual testing process stays pretty much the same. So what is it that will make you stand out?
+
+Here at OmbuLabs we have some techniques that we employ that ensure our high satisfaction rates. In this post, we’ll share some tips with you.
+
+<!--more-->
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">A QA engineer walks into a bar. Orders a beer. Orders 0 beers. Orders 99999999999 beers. Orders a lizard. Orders -1 beers. Orders a ueicbksjdhd. <br><br>First real customer walks in and asks where the bathroom is. The bar bursts into flames, killing everyone.</p>&mdash; Brenan Keller (@brenankeller) <a href="https://twitter.com/brenankeller/status/1068615953989087232?ref_src=twsrc%5Etfw">November 30, 2018</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+##So… How do I become a better Manual Tester?
+
+Anyone with common sense, empathy and basic knowledge of computer applications can run manual tests. But to ensure quality tests are being run and quality bug reports are being produced, there are certain things that need to be observed. 
```

Comment:
Maybe remove "common sense, empathy" and let just " basic knowledge of computer" it might sound rude?

I know some old people that have common sense and are emphatic but I bet they could could not test an application. 

![91sn32Q](https://user-images.githubusercontent.com/4595174/65849556-4fd34480-e308-11e9-99b0-708cbd0795ec.jpg)

Reply:
Haha true. But, similarly, there are people with basic computer knowledge that wouldn't make good manual testers. Common sense and empathy are important when thinking of things a user might do or go through when using the app.